modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/DropIndexEntry.java

```diff
@@ -33,7 +33,7 @@
  * the {@link CatalogIndexStatus#STOPPING} state.
  */
 public class DropIndexEntry extends AbstractChangeIndexStatusEntry implements Fireable {
-    public static final DropIndexEntrySerializer SERIALIZER = new DropIndexEntrySerializer();
+    public static final CatalogObjectSerializer<DropIndexEntry> SERIALIZER = new DropIndexEntrySerializer();
 
     private final int tableId;
```

Comment:
looks like `tableId` is not used anywhere but tests. Let's remove it as well

Reply:
Fixed.